js/main-hovers.js

```diff
@@ -0,0 +1,156 @@
+// base elements
+const body = document.getElementById('arrow-main');
+const baseArrowLogo = document.getElementById('base-arrow-animation');
+const incubatorBaseLogo = document.getElementById('incubator-base-animation');
+const navBrandWhite = document.getElementById('nav-brand-white');
+const navBrandDark = document.getElementById('nav-brand-dark');
+const siteFooter = document.getElementById('site-footer');
+const siteNav = document.getElementById('site-nav');
+const navIconOpenWhite = document.getElementById('nav-icon-open-white');
+const navIconOpenDark = document.getElementById('nav-icon-open-dark');
+
+
+// core elements
+const coreArrowLogo = document.getElementById('core-arrow-animation');
+const incubatorCoreLogo = document.getElementById('incubator-core-animation');
+
+// fx elements
+const fxArrowLogo = document.getElementById('fx-arrow-animation');
+const incubatorFxLogo = document.getElementById('incubator-fx-animation');
+
+// optics elements
+const opticsArrowLogo = document.getElementById('optics-arrow-animation');
+const incubatorOpticsLogo = document.getElementById('incubator-optics-animation');
+
+// incubator elements
+const incubatorArrowLogo = document.getElementById('logo-white-lines');
+const incubatorLinks = document.querySelectorAll('.incubator-items a');
+const incubatorHoverLogo = document.getElementById('incubator-hover-animation');
+const incubatorList = document.getElementById('incubator-list');
+
+// features/categories elements
+const categoryRow = document.querySelectorAll('#main-flex div');
+const categoryIconWhite = document.getElementsByClassName('cat-icon-white');
+const categoryIconDark = document.getElementsByClassName('cat-icon-dark');
+const catIconCoreColor = document.getElementById('cat-icon-color-core');
+const catIconFxColor = document.getElementById('cat-icon-color-fx');
+const catIconOpticsColor = document.getElementById('cat-icon-color-optics');
+
+function checkActiveFeature() {
+
+  Array.prototype.map.call(arrowFeatures, function(el) {
+
+    const current_id = el.id;
+    const current_class = el.classList;
+
+    if(current_class.contains('active')) {
+      switch (current_id) {
+        case 'core':
+          commonHoverStyle(current_id);
+          coreHoverStyle();
+          break;
+        case 'fx':
+          commonHoverStyle(current_id);
+          fxHoverStyle();
+          break;
+        case 'optics':
+          commonHoverStyle(current_id);
+          opticsHoverStyle();
+          break;
+        case 'incubator':
+          commonHoverStyle(current_id, true);
+          incubatorHoverStyle();
+          break;
+        default:
+          baseHoverStyle();
+      }
+    }
+  });
+}
+
+function setOpacity(iconElements, id, opacity) {
+  Array.prototype.map.call(iconElements, function(obj) {
```

Comment:
This is a extremely weird way of calling a function :man_shrugging

```suggestion
  Array.map(iconElements, function(obj) {
```